kubernetes/openshift-example/templates/template_patroni_persistent.yaml

```diff
@@ -106,6 +106,20 @@ objects:
           application: ${APPLICATION_NAME}
           cluster-name: ${PATRONI_CLUSTER_NAME}
       spec:
+        initContainers:
+        - command:
+          - sh
+          - -c 
+          - "mkdir -p /home/postgres/pgdata/pgroot/data && chmod 0700 /home/postgres/pgdata/pgroot/data"
```

Comment:
I would suggest to create/assign permissions only for `/home/postgres/pgdata/pgroot`

Reply:
I tried but it keeps complaining about permissions of the ./data directory. :-/

`2020-01-20 10:35:25.972 UTC [47] FATAL:  data directory "/home/postgres/pgdata/pgroot/data" has invalid permissions`